index.md

```diff
@@ -5,137 +5,143 @@ layout: default
 # Salt
 > Looking at big data? Add a little salt.
 
-## Usage
+Displaying a massive dataset is challenging. Your visualization must summarize all the data at a high level and still be able to reveal localized trends among billions of data points. But structuring your data to meet one of these requirements often precludes the ability to fulfill the other. And in either case, processing the data can consume significant time and resources.
```

Comment:
Let's embed the torque example right at the top of the page, and alter these paragraphs to speak about Salt from the perspective of that concrete example.